package.json

```diff
@@ -70,6 +70,7 @@
   },
   "typings": "index.d.ts",
   "devDependencies": {
+    "@types/node": "^14.0.10",
```

Comment:
we use NodeJS namepace in declaring ISuitestBase

Reply:
OK, I'll revert this one. But in general, it's still dev dependencies. In production JS code does not care about TypeScript interface